js/components/Leaderboard/index.js

```diff
@@ -21,9 +21,6 @@ class Leaderboard extends React.Component {
 			filter: {
 				campaign_id: '7536',
 			},
-      filter: {
```

Comment:
Ahhh, that's why there was no `campaign_id` filter. We were overwriting it! 🙃

Reply:
that is what I was wondering too!